crates/cargo-bsp/src/project_model/mod.rs

```diff
@@ -8,3 +10,12 @@ pub(crate) mod project_manifest;
 pub(crate) mod sources;
 pub(crate) mod target_details;
 pub mod workspace;
+
+pub trait DefaultFeature {
```

Comment:
czemu to jest trait w sumie?

Reply:
Oj, to wynikło z propozycji Julii, żeby zrobic to jako implementacje Default traita. Już poprawiam

Reply:
A jednak nie. Chodziło o to, że nie można zdefiniować impl blocku dla struktury zdefiniowanej w innym package'u